### PR TITLE
Hfeyp 597 remove back button on edit page screen

### PR DESCRIPTION
**Pull request by agoldstone93**
## Ticket and context

Ticket: [HFEYP-597](https://dfedigital.atlassian.net/browse/HFEYP-597)
Removed the back button from the edit published and draft pages. Cancel button remains.
### Code quality checks
- [ ] All commit messages are meaningful and true

### How can someone see it in review app?
1. Click the link to review app and go to `/cms/pages` [or click this link](https://eyfs-cms-review-pr-530.london.cloudapps.digital/cms/pages)
2. Choose a page and click `versions`
3. Click `edit` next to a live page
4. Add some content
5. Click `cancel` at the bottom. Verify that there is a dialog box to confirm the cancellation. Verify that the edits are removed and the page reloads.

### CODE REVIEW

**Review by reggieb — APPROVED** [review]
The code change does what the ticket requires.

Though it's weird behaviour. I'd have expected cancel to take me to another page (the show action for example). You have to click Save to get to to show even though there are no changes, or use one of the navigation link to leave the page. However, that's out of scope of this ticket.

File: app/views/content_page_versions/_form.html.erb [hunk 1]
```diff
@@ -26,8 +26,7 @@
           <% if action_name === "edit" || action_name === "new" || action_name === "update" %>
             <div class="fixed-footer">
               <%= form.govuk_submit "Save as draft" %>
-              <%= link_to 'Cancel', edit_content_page_content_page_version_path(@content_page_version.content_page, @content_page_version), :data => {:confirm => "Are you sure you want to undo your changes ?", :class => "govuk-link"}%> |
-              <%= link_to 'Back', versions_content_page_path(@content_page_version.content_page), :data => {:confirm => "Any changes will be lost, are you sure ?", :class => "govuk-link"}%>
+              <%= link_to 'Cancel', edit_content_page_content_page_version_path(@content_page_version.content_page, @content_page_version), data: { confirm: 'Are you sure you want to undo your changes ?' }, class: "govuk-link" %>
```
[the remark this answers is not in the document]
Comment (agoldstone93) [reply app/views/content_page_versions/_form.html.erb:29]: I think it was the webpack issue actually, but I moved `class` anyway whilst I was at it

[the remark this answers is not in the document]
Comment (reggieb) [reply app/views/content_page_versions/_form.html.erb:29]: Ah, yes. If webpacker wasn't loading, JavaScript wouldn't load properly, so the pop-up dialogue wouldn't work.

**Comment by ebrett** [comment]
I don't think this go in as the behaviour @reggieb describes above is strange and counterintuitive.  I'd rather keep the back button and remove the 'Cancel' button.  I think when the original ticket was written the two did the same thing (or at least we thought they did).  But the cancel merely refreshes the page (losing any changes) and comes back to the edit page, where as the back button went back to the index page (for the versions in this case).

**Comment by reggieb** [comment]
@ebrett You're right. I'd suggest removing the cancel button and relabelling the Back button as Cancel.

**Comment by agoldstone93** [comment]
I've made the suggested changes so the button now says cancel and returns to the previous screen

**Review by ebrett — APPROVED** [review]
Much better.

File: app/views/content_page_versions/_form.html.erb [hunk 2]
```diff
@@ -26,8 +26,7 @@
           <% if action_name === "edit" || action_name === "new" || action_name === "update" %>
             <div class="fixed-footer">
               <%= form.govuk_submit "Save as draft" %>
-              <%= link_to 'Cancel', edit_content_page_content_page_version_path(@content_page_version.content_page, @content_page_version), :data => {:confirm => "Are you sure you want to undo your changes ?", :class => "govuk-link"}%> |
-              <%= link_to 'Back', versions_content_page_path(@content_page_version.content_page), :data => {:confirm => "Any changes will be lost, are you sure ?", :class => "govuk-link"}%>
+              <%= link_to 'Cancel', versions_content_page_path(@content_page_version.content_page), data: { confirm: "Any changes will be lost, are you sure ?"}, class: "govuk-link" %>
```
Comment (reggieb) [inline app/views/content_page_versions/_form.html.erb:29]: Really picky - but there should be a space at the end of the hash (before the closing curly bracket).
```suggestion
              <%= link_to 'Cancel', versions_content_page_path(@content_page_version.content_page), data: { confirm: "Any changes will be lost, are you sure ?" }, class: "govuk-link" %>
```

**Pull request merged**